Android debug build > :app:bundleDebugJsAndAssets_SentryUpload FAILED

OS:
- [ ]  Windows
- [X]  MacOS
- [ ]  Linux

_Platform:_
- [ ]  iOS
- [X]  Android

```
> Task :app:bundleDebugJsAndAssets_SentryUpload FAILED
  INFO    2019-01-31 15:57:24.025993 +01:00 Loaded config from /Users/brian/.sentryclirc
  DEBUG   2019-01-31 15:57:24.027105 +01:00 sentry-cli version: 1.37.4, platform: "darwin", architecture: "x86_64"
  INFO    2019-01-31 15:57:24.027133 +01:00 sentry-cli was invoked with the following command line: "/Users/brian/punten-app/node_modules/@sentry/cli/sentry-cli" "--log-level" "debug" "react-native" "gradle" "--bundle" "/Users/brian/punten-app/android/app/build/generated/assets/react/debug/index.android.bundle" "--sourcemap" "/Users/brian/punten-app/android/app/build/generated/assets/react/debug/index.android.bundle.map" "--release" "nl.puntje.business-1.0.8" "--dist" "8"
  INFO    2019-01-31 15:57:24.030271 +01:00   bundle path: /Users/brian/punten-app/android/app/build/generated/assets/react/debug/index.android.bundle
  INFO    2019-01-31 15:57:24.030290 +01:00   sourcemap path: /Users/brian/punten-app/android/app/build/generated/assets/react/debug/index.android.bundle.map
Processing react-native sourcemaps for Sentry upload.
> Analyzing 2 sources
  DEBUG   2019-01-31 15:57:24.031403 +01:00 error: running update nagger
  DEBUG   2019-01-31 15:57:24.031692 +01:00 skipping update nagger because session is not attended
error: No such file or directory (os error 2)
  DEBUG   2019-01-31 15:57:24.037634 +01:00 client close; no transport to shut down  (from sentry)
```

Config:

index.js
```
Sentry.config('https://...@sentry.io/...').install()
```



---
I have following issue:

I am getting a build error when running `react-native run-android`

**Steps to reproduce:**
- Install react native sentry
- Link it
- Run react-native run-android
- Getting `ask :app:bundleDebugJsAndAssets_SentryUpload FAILED` error.

Why does it attempt to upload source map for debug build? Can I disable this?

same here, bundleInDebug: true not works for me, getting the same output

```
> Task :app:bundleDebugJsAndAssets_SentryUpload
Processing react-native sourcemaps for Sentry upload.
> Analyzing 2 sources
> Rewriting sources
> Adding source map references
error: project not found

Add --log-level=[info|debug] or export SENTRY_LOG_LEVEL=[info|debug] to see more output.
Please attach the full debug log to all bug reports.


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:bundleDebugJsAndAssets_SentryUpload'.
> Process 'command 'node_modules/@sentry/cli/bin/sentry-cli'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 34s
76 actionable tasks: 2 executed, 74 up-to-date
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/getting-started.html
```

--- a/sentry.gradle
+++ b/sentry.gradle
@@ -11,7 +11,7 @@ gradle.projectsEvaluated {
     // separately we then hook into the bundle task of react native to inject
     // sourcemap generation parameters.  In case for whatever reason no release
     // was found for the asset folder we just bail.
-    def bundleTasks = tasks.findAll { task -> task.name.startsWith("bundle") && task.name.endsWith("JsAndAssets") }
+    def bundleTasks = tasks.findAll { task -> task.name.startsWith("bundle") && task.name.endsWith("JsAndAssets") && !task.name.contains("Debug")}
     bundleTasks.each { bundleTask ->
         def shouldCleanUp
         def sourcemapOutput

